src/app/layout.tsx

```diff
@@ -1,4 +1,5 @@
 import type { Metadata } from "next";
+import type { Viewport } from "next";
```

Comment:
These should be able to be combined into one

Reply:
I fully expected Prettier to fix that :P